apps/pie-storybook/stories/pie-cookie-banner.stories.ts

```diff
@@ -30,6 +37,24 @@ const cookieBannerStoryMeta: CookieBannerStoryMeta = {
 
 export default cookieBannerStoryMeta;
 
+const BaseStoryTemplate = (props: CookieBannerProps) : TemplateResult => html`
+        <pie-cookie-banner></pie-cookie-banner>`;
+
+/**
+ * Creates a 'page' of scrollable HTML. Useful for testing scroll behaviours in a Story.
+ */
+const createScrollablePageHTML = () : TemplateResult => {
```

Comment:
I would probably use the [lorem-ipsum package](https://www.npmjs.com/package/lorem-ipsum) for this (and any other faux text use-cases we have).

Happy for that to be done separately to replace it everywhere thought potentially (maybe add a TODO here if others agree)